wicket-native-websocket/wicket-native-websocket-javax/src/main/java/org/apache/wicket/protocol/ws/javax/JavaxUpgradeHttpRequest.java

```diff
@@ -77,6 +78,7 @@ public JavaxUpgradeHttpRequest(final Session session, EndpointConfig endpointCon
 		this.userPrincipal = session.getUserPrincipal();
 		Object requestURI = session.getRequestURI();
 		this.requestUri = requestURI != null ? requestURI.toString() : "";
+        this.contextPath = httpSession.getServletContext().getContextPath();
```

Comment:
Hum... locally it looks Ok :-(

Reply:
You are using spaces and it should be tabs :)
You can turn on display of hidden chars .. :))

Reply:
Thanks! Fixed